src/lib/rest/uriParamNames.h

```diff
@@ -45,6 +45,7 @@
 #define URI_PARAM_ATTRIBUTE_FORMAT        "attributeFormat"
 #define URI_PARAM_OPTIONS                 "options"
 #define URI_PARAM_TYPE                    "type"
+#define URI_PARAM_SORTED                  "sorted"
```

Comment:
Adjectives are normally used for booleans, like *occupied*, or *empty*.
You don't expect them to carry any value.

I vote for sortBy or orderBy, both excellent choices.
ORDER BY is used in SQL and is probably familiar to a lot of our users.